shared/schemas/workflow-options-output-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Workflow Results",
+    "description": "Results of a workflow execution",
+    "type": "object",
+    "properties": {
+        "completedWith": {
+            "description": "The state of workflow completion.",
+            "type": "string",
+            "enum": [
+                "error",
+                "success"
+            ]
+        },
+        "description": {
+            "description": "High-level summary of the current status, free-form text, human readable.",
+            "type": "string"
+        },
+        "nextWorkflows": {
+            "description": "List of workflows suggested to run next. Items at lower indexes are of higher priority.",
+            "type": "array",
+            "items": {
+                "$ref": "workflow-ref-schema.json"
+            }
+        },
+        "options": {
```

Comment:
I don't understand what is this field/object for. if it is for an unknown piece of data then I suggest removing it. the results should be a well-defined object.

Reply:
M2K has no real output per se: the questionnaire link is useful during the execution but in the end, the output of M2K is what is in the target git repo and branch which is provided as input

Reply:
the exising examples are all links. so we do see there's a unified format. we can limit the output we expect. we can say that the output will have messages and links and that's it

Reply:
Per offline discussion, a list with priorities is what we introduce.
In the future, we might add a field to clasify the suggested workflows (like recommended, optional, etc)